libs/vertexai/tests/unit_tests/test_embeddings.py

```diff
@@ -8,6 +8,16 @@
 from langchain_google_vertexai.embeddings import GoogleEmbeddingModelType
 
 
+def test_init() -> None:
+    for embeddings in [
+        VertexAIEmbeddings(
```

Comment:
hm seems instantiating vertex embedding model requires setting up vertex creds which isn't great. is there a way to change that in a non-breaking fashion @lkuligin?

Reply:
ideally it should be supported on the SDK side, I've raised a FR.

Reply:
so far I'd suggest let's move this to integration tests, wdyt?

Reply:
moved 👍